Makefile

```diff
@@ -108,24 +109,9 @@ $(BUILD_TARGETS): go.sum $(BUILDDIR)/
 $(BUILDDIR)/:
 	mkdir -p $(BUILDDIR)/
 
-build-reproducible: go.sum
```

Comment:
this really useful have a command to do reproducible builds

Reply:
we also need  to expand on this based on
https://github.com/cosmos/gaia/pull/523